### ISSUE TITLE
squiggle not showing in Edge browser

### ISSUE THREAD

**Issue by colombod** [Microsoft/monaco-editor#764]
monaco-editor version: 0.11.0
Browser: Microsoft Edge 41.16299.248.0
OS: WIN 10 1809#
Go to https://try.dot.net and paste in the following:
```Console``` and click the Run button.

In chrome, a red squiggle appears on the text.
In edge, no squiggle appears (see screenshot)

Steps or JS usage snippet reproducing the issue:
enter an invalid c# code when diagnostics are reported no red squiggle appears on screen

**Comment by alexandrudima** [comment]
@colombod When I go to https://try.dot.net/ and I go to the `Sources` tab and look for `editor.main.js` I see the following:

```
/*!-----------------------------------------------------------
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Version: 0.10.0(9b725bd7807ee959240b4af7906fae119b0fbf9e)
 * Released under the MIT license
 * https://github.com/Microsoft/vscode/blob/master/LICENSE.txt
 *-----------------------------------------------------------*/
```
This makes me believe that you are trying with monaco-editor version 0.10.0 and not 0.11.0

---

I believe you have forgotten to attach a screenshot.

---

If I go to  https://try.dot.net using Edge and I open F12 (the equivalent Developer Tools for Edge) I see the following warnings/errors:

![image](https://user-images.githubusercontent.com/5047891/37454265-65116faa-283a-11e8-9756-087f4766f745.png)

If I do ctrl+F in the editor I can see that all of the editor images are missing:
![image](https://user-images.githubusercontent.com/5047891/37454307-804aa624-283a-11e8-8aa2-ad9ecc106e71.png)

---

**TL;DR** You are using monaco editor 0.10.0 and have a configuration problem with `Content-Security-Policy`

**Comment by colombod** [comment]
I forgot to mention that the live version of our site is still on the old version of Monaco Editor. Will attach screenshot then

**Comment by alexandrudima** [comment]
Please read my comment again:

I believe you have a configuration problem with `Content-Security-Policy`. Icons appear for me when I use Edge to visit https://microsoft.github.io/monaco-editor/
